webpack/assets/javascripts/react_app/components/common/emptyState/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { EmptyState as PfEmptyState, Button } from 'patternfly-react';
+
+const EmptyState = (props) => {
```

Comment:
Need testing?

Reply:
Yep, I might miss the tests when I moved this component from `Katello`.
Thanks for noticing :100:

Reply:
Sorry no tests exist in `Katello` for this component, I will add some in the next days.